src/en/hentaidexy/src/eu/kanade/tachiyomi/extension/en/hentaidexy/Hentaidexy.kt

```diff
@@ -63,7 +61,7 @@ class Hentaidexy : HttpSource() {
 
     // latest
     override fun latestUpdatesRequest(page: Int): Request {
-        return GET("$apiUrl/api/v1/mangas?page=$page&limit=100&sort=-updatedAt", headers)
+        return GET("$baseUrl/api/mangas?page=$page&limit=100&sort=-createdAt", headers)
```

Comment:
Please confirm that this is correct. 

Usually 'created' is sorted by when an entry was added. 
And 'updated' is usually sorted by when the latest chapter was added. (This is preferred)